functional/OpenJcePlusTests/build.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0"?>
+
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<project name="OpenJcePlus tests" default="build" basedir=".">
+	<taskdef resource="net/sf/antcontrib/antlib.xml" />
+	<description>
+		OpenJCEPlus Tests
+	</description>
+
+	<!-- set global properties for this build -->
+	<property name="DEST" value="${BUILD_ROOT}/functional/OpenJcePlusTests" />
+
+	<!--Properties for this particular build-->
+	<property name="src" location="./OpenJCEPlus/src/" />
+	<property name="build" location="./bin" />
+	<property name="LIB" value="junit4,hamcrest_core,bcprov_jdk18on"/>
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
+
+	<target name="openJcePlusTests.check">
+		<condition property="openJcePlusTests.exists">
+			<available file="OpenJCEPlus" type="dir"/>
+		</condition>
+	</target>
+
+	<if>
+		<contains string="${SPEC}" substring="zos"/>
+		<then>
+			<property name="GIT_REPO" value="git@github.com:" />
+		</then>
+		<else>
+			<property name="GIT_REPO" value="https://github.com/" />
+		</else>
+	</if>
+	<target name="getOpenJcePlusTests" depends="openJcePlusTests.check" unless="openJcePlusTests.exists">
+		<exec executable="git" failonerror="true">
+			<arg value="clone" />
+			<arg value="${GIT_REPO}IBM/OpenJCEPlus.git" />
+		</exec>
+		<delete dir="./OpenJCEPlus/src/main" />
+	</target>
+
+	<target name="init">
+		<mkdir dir="${DEST}" />
+		<mkdir dir="${build}" />
+	</target>
+
+	<property name="addExports" value='--add-exports java.base/sun.security.internal.spec=openjceplus --add-exports java.base/sun.security.util=openjceplus --add-exports java.base/sun.security.x509=openjceplus --add-exports java.base/sun.security.pkcs=openjceplus --add-exports java.base/sun.security.internal.interfaces=openjceplus --add-exports java.base/sun.util.logging=openjceplus --add-exports java.base/jdk.internal.logger=openjceplus --add-exports openjceplus/com.ibm.crypto.plus.provider.ock=ALL-UNNAMED --add-exports java.base/sun.security.util=ALL-UNNAMED --add-exports java.base/sun.security.x509=ALL-UNNAMED --add-exports openjceplus/com.ibm.misc=ALL-UNNAMED --add-exports java.base/sun.security.pkcs=ALL-UNNAMED' />
```

Comment:
Suggest to add exports to multiple modules for the same package - `--add-exports java.base/sun.security.util=openjceplus,ALL-UNNAMED`

Reply:
Thanks Jason.